conf/app.routes

```diff
@@ -102,8 +102,10 @@ POST        /business-partners/corporate-body-vat-registration-number          @
 GET         /business-partners/corporate-body-corporation-tax-unique-taxpayer-reference @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyUtrController.load()
 POST        /business-partners/corporate-body-corporation-tax-unique-taxpayer-reference @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyUtrController.next()
 
-GET         /business-partners/corporate-body-trading-name               @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.load()
-POST        /business-partners/corporate-body-trading-name              @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.next()
+GET         /business-partners/corporate-body-trading-name                      @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.load(index: Int, mode: Mode = NormalMode)
+POST        /business-partners/corporate-body-trading-name                      @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.next(index: Int, mode: Mode = NormalMode)
+GET         /business-partners/change-corporate-body-trading-name               @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.load(index: Int, mode: Mode = CheckMode)
+POST        /business-partners/change-corporate-body-trading-name               @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersCorporateBodyTradingNameController.next(index: Int, mode: Mode = CheckMode)
```

Comment:
These still need to be updated to 
`/change-business-partners/corporate-body-trading-name`
rather than
`/business-partners/change-corporate-body-trading-name`